### ISSUE TITLE
LT in literal replaced by < in display of query results (should be easy to fix)

### ISSUE THREAD

**Issue by hannahbast** [jbuerklin/qleverUI#26]
Consider the following query on http://qlever.informatik.uni-freiburg.de/OSM_Germany, which produces a long list of results, where the last column contains literals ob type geo:wktLiteral starting with MULTIPOLYGON. The details of the query do not matter.

The literals are all displayed as MU because the LT is replaced by < (apparently because it is taken as or replaced by &lt;). This is of course wrong.

```
PREFIX geo: <http://www.opengis.net/ont/geosparql#>
PREFIX osmkey: <https://www.openstreetmap.org/wiki/Key:>
SELECT ?x ?name ?waterway ?hasgeometry WHERE {
  ?x osmkey:name ?name .
  ?x osmkey:waterway ?waterway .
  ?x geo:hasGeometry ?hasgeometry .
}
```

**Comment by hannahbast** [comment]
Thanks for the fix of #27 :-)

Any chance to get a quick fix for this issue, too? It looks rather trivial to me, but is rather annoying in its current state

**Comment by jbuerklin** [comment]
I wasn't really quick at it, but I fixed it, eventually.
